Find a functionally equivalent code:android.content.res.AssetManager.createTheme:COMMENT
Method Modifier: default     final       
Comment:/*package*/

@@ -3,9 +3,9 @@
         if (!mOpen) {
             throw new RuntimeException("Assetmanager has been closed");
         }
-        int res = newTheme();
+        long res = newTheme();
         incRefsLocked(res);
         return res;
     }

